Shokofin/Configuration/configPage.html

```diff
@@ -290,7 +290,11 @@ <h3>Advanced Settings</h3>
                         </div>
                         <div class="inputContainer inputContainer-withDescription">
                             <input is="emby-input" type="text" id="IgnoredFileExtensions" label="Ignored file extensions:" />
-                            <div class="fieldDescription">A space seperated list of file extensions which will be ignored during the library scan.</div>
+                            <div class="fieldDescription">A space separated list of file extensions which will be ignored during the library scan.</div>
+                        </div>
+                        <div class="inputContainer inputContainer-withDescription">
```

Comment:
We either need a new separate filter function for the folders or to make the ignored extensions a comma only separated list.

Reply:
made a separate filter function